File: Modul-3/Soal-2/PRAK302-2410817220030-RachelWinaYuda.py
nilai = int(input(""))

if nilai >= 80:
    grade = 'A'
elif nilai >= 70:
    grade = 'B'
elif nilai >= 60:
    grade = 'C'
elif nilai >= 50:
    grade = 'D'
else:
    grade = 'E'

print(f"{grade}")

File: Modul-3/Soal-2/PRAK302-2410817220030-RachelWinaYuda.c
#include <stdio.h>

int main() {
    int nilai;
    char grade;
    printf("");
    scanf("%d", &nilai);
    if (nilai >= 80){grade = 'A';} 
    else if (nilai >= 70){grade = 'B';} 
    else if (nilai >= 60){grade = 'C';}
    else if (nilai >= 50){grade = 'D';}
    else {grade = 'E';}
    printf("%c\n", grade);
    return 0;
}